dataminer-overview/Legal/DaaS_Planned_Maintenance_Policy.md

```diff
@@ -0,0 +1,98 @@
+---
+uid: DaaS_Planned_Maintenance_Policy
+---
+
+# DaaS (DataMiner as a Service) Planned Maintenance Policy
+
+This policy outlines the procedures and guidelines for planned maintenance on DaaS (DataMiner as a Service). As part of our commitment to providing a secure, reliable, and high-performing service, we will conduct regular maintenance to apply system updates and security patches. This policy ensures that maintenance activities are predictable, transparent, and minimally disruptive to our users.
+
+## Maintenance window
+
+Planned maintenance will be conducted monthly during a predefined maintenance window.
+
+- Scheduled maintenance window: During the second week of each month.
+
+## Scope of maintenance
+
+The monthly maintenance will include the following activities:
+
+- Operating system updates: Applying the latest operating system security patches and updates to ensure a secure environment.
+
+- Security patches: Implementation of critical security patches to mitigate vulnerabilities and potential threats.
+
+- DataMiner software updates:
+
+  - Main Release (current CU): The DataMiner software will be updated to the latest Cumulative Update (CU).
+  - Feature Release (latest version): For users subscribed to the feature release, we will update to the latest available feature release version of DataMiner.
+
+## Communication
+
+We will ensure transparent communication regarding maintenance activities, including any potential system unavailability:
+
+- Notification in advance: Users will receive a maintenance notification at least [X days/weeks] in advance of the scheduled maintenance window. This notification will include details such as:
```

Comment:
```suggestion
- Notification in advance: In case any downtime or disruption is expected outside of the regular monthly update routine, users will receive a maintenance notification at least 1 month in advance of the scheduled maintenance window. This notification will include details such as:
```